File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        
        int[] list = new int[n + 1];
        int[] res  = new int[n + 1];

        for ( int i = 0; i <=n; i++ ) {
            list[i] = i;
        }

        for (int num : list ) {
            int counter = 0;
            int index   = num;
            while ( num != 0 ) {
                if ( (num & 1) == 1 ) counter++;
                num >>= 1;
            }
            res[index] = counter;
        }
        return res;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        
        res = []
        counter = defaultdict()

        for n in nums:
            counter[n] = counter.get(n, 0) + 1

        for it in counter:
            
            if counter[it] > len(nums) // 3:
                res.append(it)

        return res


